mws/utils.py

```diff
@@ -279,6 +279,23 @@ def get_utc_timestamp():
     return datetime.datetime.utcnow().replace(microsecond=0).isoformat()
 
 
+def validate_package_update_details(package_update_details):
```

Comment:
This function belongs in `apis/easyship.py`, as it is specific to that module.

Other than that, the content of the function looks good.